system-contracts/test/PubdataChunkPublisher.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { expect } from "chai";
+import { ethers, network } from "hardhat";
+import type { Wallet } from "zksync-web3";
+import type { PubdataChunkPublisher } from "../typechain";
+import { PubdataChunkPublisherFactory } from "../typechain";
+import { TEST_L1_MESSENGER_SYSTEM_CONTRACT_ADDRESS, TEST_PUBDATA_CHUNK_PUBLISHER_ADDRESS } from "./shared/constants";
+import { prepareEnvironment } from "./shared/mocks";
+import { deployContractOnAddress, getWallets } from "./shared/utils";
+
+describe("PubdataChunkPublisher tests", () => {
+  let wallet: Wallet;
+  let l1MessengerAccount: ethers.Signer;
+
+  let pubdataChunkPublisher: PubdataChunkPublisher;
+
+  const genRandHex = (size) => ethers.utils.hexlify(ethers.utils.randomBytes(size));
```

Comment:
I think you can just use `ethers.utils.randomBytes` everywhere. I think that ethers.js should be fine parsing array of bytes without it being a hex string